### ISSUE TITLE
Highlight Matching Tags keeps re-enabling

### ISSUE THREAD

**Issue by nunoperalta** [notepad-plus-plus/notepad-plus-plus#4958]
### Description of the Issue
Every time I reopen Notepad++, this setting seems to reset. I do NOT want this whole setting enabled.

![image](https://user-images.githubusercontent.com/487079/47454253-af56d500-d7c6-11e8-9f5a-5e6cf1a7a81c.png)

### Expected Behavior
![image](https://user-images.githubusercontent.com/487079/47454290-c4336880-d7c6-11e8-911e-07fc357fe14c.png)

### Actual Behavior
![image](https://user-images.githubusercontent.com/487079/47454253-af56d500-d7c6-11e8-9f5a-5e6cf1a7a81c.png)

### Debug Information
Notepad++ v7.5.9   (32-bit)
Build time : Oct 14 2018 - 15:02:52
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 10 (64-bit)
Plugins : mimeTools.dll NppConverter.dll NppExport.dll ComparePlugin.dll

**Comment by cmeriaux** [comment]
I can't reproduce your issue with the same version of Notepad++.
Your configuration files may be corrupted, and Notepad++ encountered a silent exception while loading or storing the configuration.
Try with the zip package https://notepad-plus-plus.org/repository/7.x/7.5.9/npp.7.5.9.bin.zip 
Try with a fresh install config.

**Comment by nunoperalta** [comment]
Ok. Doesn't seem to happen anymore.

The thing is... a few days ago, I had to open Notepad++ as Administrator. I did it manually by right-clicking in the Task Bar and then "Run as Administrator".

That seems to have completely reset ALL the configs.

When I re-opened Notepad++ as normal user, I put all my settings back to normal and actually made a backup of the "config.xml".

After a few restarts/reboots, etc, it's been already the 3rd time I had to disable that config.


I updated to 7.5.9 three or four days ago. I was a few versions behind, but definitely this has never happened to me before that version.

Somehow I don't seem to reproduce this anymore. I tried to run as admin, restart, etc... everything seems to be ok.

I'll let you know if I find exactly how to reproduce this. For now it appears to be a bit random.
